azerothcore/azerothcore-wotlk
# feat(Scripts/Commands): Implement restart/shutdown reasons

## Changes Proposed:
-  Add reason param to the server restart/shutdown commands
-  

## Issues Addressed:

- Closes 

## SOURCE:

TrinityCore

## Tests Performed:

- Tested ingame
- 


## How to Test the Changes:


1.Log a character
2. .server restart 120 updates
3. Check that it displays reason ingame

## Known Issues and TODO List:


- [ ] Commands can't take more than a single word as param


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
